apps/src/applab/ApplabVisualizationColumn.jsx

```diff
@@ -12,32 +23,49 @@ var connect = require('react-redux').connect;
 var ApplabVisualizationColumn = React.createClass({
   propTypes: {
     isReadOnlyWorkspace: React.PropTypes.bool.isRequired,
+    hideSource: React.PropTypes.bool.isRequired,
+    isEmbedView: React.PropTypes.bool.isRequired,
+
     isEditingProject: React.PropTypes.bool.isRequired,
     screenIds: React.PropTypes.arrayOf(React.PropTypes.string).isRequired,
     onScreenCreate: React.PropTypes.func.isRequired,
   },
 
   render: function () {
+    var showFrame = !this.props.isShareView;
+
     return (
-      <div id="visualizationColumn">
+      <div
+          id="visualizationColumn"
+          className={classNames({with_padding: this.props.visualizationHasPadding})}
+          style={[
+            (this.props.isEmbedView || this.props.hideSource) && styles.nonResponsive
+          ]}
```

Comment:
I haven't really decided whether I prefer arrays of styles inlined like this, or to pre-declare a variable and just do `style={vizStyle}`. I think I might have a slight preference for how I've done it here, especially for elements whose main purpose is styling (such as this div). Thoughts?